stack.yaml

```diff
@@ -21,6 +21,7 @@ extra-deps:
 - teardown-0.5.0.0
 - unliftio-0.2.7.0
 - yaml-0.8.32
+- safe-exceptions-0.1.7.0
```

Comment:
Universum reexports `safe-exceptions` too, why did you choose this particular version?